advisories/github-reviewed/2023/05/GHSA-jh85-wwv9-24hv/GHSA-jh85-wwv9-24hv.json

```diff
@@ -20,13 +20,6 @@
         "ecosystem": "PyPI",
         "name": "pymdown-extensions"
       },
-      "ecosystem_specific": {
```

Comment:
Why is this information removed?

Reply:
Oh! Good question, and sorry to have missed that. I thought I ONLY changed the title.

Reply:
Did my change go against the first revision from https://github.com/github/advisory-database/commits/main/advisories/github-reviewed/2023/05/GHSA-jh85-wwv9-24hv/GHSA-jh85-wwv9-24hv.json and not the second version?

Reply:
I guess? I'm not sure, but it would appear so.

Reply:
I'll close this and redo it. No reason to ask anyone else to do that work for me!